mayadata-io/gui-automation
# GUI Teaming test cases

- ProjectOwner can invite others in his/her Project
-  To view pending invites
-  Only invitee can accept or reject invitation
-  Validate invitation reject process
-  Validate invitation accept process
-  Project owner can only change the member role
-  Non Project owner can not change the member role
-  To verify Project Owner access
-  To verify Project Admin access
-  To verify Project Read Admin access
-  To verify Project Member access
- Readme file for teaming test cases
